grapher/core/Grapher.tsx

```diff
@@ -2529,17 +2526,6 @@ export class Grapher
         this.analytics.logGrapherTimelinePlay(this.slug)
     }
 
-    // todo: restore this behavior??
-    onStartPlayOrDrag(): void {
-        this.debounceMode = true
```

Comment:
We might still need this, not entirely sure. The only place it is used is here, in explorers: https://github.com/owid/owid-grapher/blob/d9b38d588566ab0bdfc93feeba42dc3e2db13df2/explorer/Explorer.tsx#L420-L422

Reply:
Thanks Marcel – good catch!